scanners/boostsecurityio/codeql/module.yaml

```diff
@@ -19,9 +19,9 @@ setup:
            exit 1;;
       esac
 
-      curl -fsSL -O "https://github.com/github/codeql-action/releases/download/codeql-bundle-20221010/codeql-bundle-linux64.tar.gz"
+      curl -fsSL -O "https://github.com/github/codeql-action/releases/download/codeql-bundle-v2.13.4/codeql-bundle-linux64.tar.gz"
```

Comment:
2.13.4 is a pre-release version. idk if they tend to change anything until a release is finalized but if they do it might suddenly break the checksum check below?